test/Test/Indigo/Contracts/HomebaseLite/Metadata/Views.hs

```diff
@@ -25,8 +23,8 @@ test_currentConfig :: TestTree
 test_currentConfig =
   testScenarioOnEmulator "currentConfig view works" $ scenarioEmulated do
     (admin, contract) <- deployContract
-    Showing <$> callOffChainView contract "currentConfig" NoParam
-      @@== Showing (sConfiguration (defaultStorage admin Nothing))
+    Showing <$> callOffChainView @Configuration contract "currentConfig" NoParam
+      @@== Showing (toRPC $ sConfiguration (defaultStorage (MkConstrainedAddress admin) Nothing))
```

Comment:
We have `toL1Address` in `Test.Cleveland.Lorentz` actually, so the same reasoning as with MkAddress vs toAddress applies.